src/widgets/VaccineChart.js

```diff
@@ -34,112 +42,214 @@ export class VaccineChart extends React.Component {
-      // minTemp default prop is Infinity.
-      minTemp !== Infinity ? (
-        <VictoryLine
-          y={() => minTemp}
-          style={{
-            data: { strokeDasharray: minLineStyles.strokeDasharray, stroke: minLineStyles.stroke },
-          }}
-        />
-      ) : null;
-
-    const maxTempPlotLine =
-      // maxTemp default prop is -Infinity.
-      maxTemp !== -Infinity ? (
-        <VictoryLine
-          y={() => maxTemp}
-          style={{
-            data: { strokeDasharray: maxLineStyles.strokeDasharray, stroke: maxLineStyles.stroke },
-          }}
-        />
-      ) : null;
-
-    const minLinePlotLine =
-      minLine.length > 0 ? (
-        <VictoryLine
-          data={minLine}
-          interpolation={minLineStyles.interpolation}
-          style={{ data: { stroke: minLineStyles.stroke } }}
-          {...dataKeys}
-        />
-      ) : null;
-
-    const minLinePlotScatter =
-      minLine.length > 0 ? (
-        <VictoryScatter
-          data={minLine}
-          style={{ data: { fill: minLineStyles.fill } }}
-          {...dataKeys}
-        />
-      ) : null;
-
-    const maxLinePlotLine =
-      maxLine.length > 0 ? (
-        <VictoryLine
-          data={maxLine}
-          interpolation={maxLineStyles.interpolation}
-          style={{ data: { stroke: maxLineStyles.stroke } }}
-          {...dataKeys}
-        />
-      ) : null;
-
-    const maxLinePlotScatter =
-      maxLine.length > 0 ? (
-        <VictoryScatter
-          data={maxLine}
-          style={{ data: { fill: maxLineStyles.fill } }}
-          {...dataKeys}
-        />
-      ) : null;
-
-    const hazardPlotScatter =
-      hazards.length > 0 ? (
-        <VictoryScatter dataComponent={<HazardPoint />} data={hazards} {...dataKeys} />
-      ) : null;
+    const { minDomain, maxDomain } = this.getDomain();
 
     return (
       <Svg>
         <VictoryChart
           width={width}
           height={height}
+          padding={{ top: 20, left: 50, right: 50, bottom: 50 }}
           style={chartStyles}
           theme={VictoryTheme.material}
           maxDomain={{ y: maxDomain }}
           minDomain={{ y: minDomain }}
         >
-          <VictoryAxis offsetY={50} />
-          <VictoryAxis dependentAxis offsetX={50} crossAxis={false} />
-          {minTempPlotLine}
-          {maxTempPlotLine}
-          {minLinePlotLine}
-          {minLinePlotScatter}
-          {maxLinePlotLine}
-          {maxLinePlotScatter}
-          {hazardPlotScatter}
+          <VictoryAxis
+            offsetY={50}
+            style={{
+              tickLabels: {
+                fontSize: axisStyles.fontSize,
+                fontFamily: axisStyles.fontFamily,
+                fill: axisStyles.fill,
+              },
+            }}
+          />
+          <VictoryAxis
+            dependentAxis
+            offsetX={50}
+            crossAxis={false}
+            tickFormat={t => `${t}℃`}
+            style={{
+              tickLabels: {
```

Comment:
change not required, could just be ticketLabels: axisStyles

Reply:
Fixed by 4a915d3.